schema.sql

```diff
@@ -0,0 +1,48 @@
+CREATE DATABASE things_records;
+
+-- Member 1 Work
+
+CREATE TABLE LABEL(
+    ID INT CONSTRAINT ID_LABEL_PK PRIMARY KEY,
+    TITLE VARCHAR,
+    COLOR VARCHAR
+);
+
+CREATE TABLE BOOKS(
+    ID INT CONSTRAINT ID_BOOKS_PK PRIMARY KEY,
+    PUBLISHER VARCHAR,
+    COVER_STATE VARCHAR,
+    PUBLISH_DATE DATE,
+    ARCHIVED BOOLEAN,
+    LABEL_ID COSTRAINT LABEL-ID_BOOKS_FK REFERENCES LABEL(ID) 
+);
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/165801100-ccd74496-55e1-4958-8f07-692e3d376b85.png)

- [ ] Kindly remove the syntax error in this code above (`CONSTRAIN` and  `LABEL_D_BOOKS_FK` also the column type`INT`). Hint :point_down: 

```rb
CREATE TABLE BOOKS(
    ID INT CONSTRAINT ID_BOOKS_PK PRIMARY KEY,
    PUBLISHER VARCHAR,
    COVER_STATE VARCHAR,
    PUBLISH_DATE DATE,
    ARCHIVED BOOLEAN,
    LABEL_ID INT CONSTRAINT LABEL_ID_BOOKS_FK REFERENCES LABEL(ID) 
);
```

Reply:
Thanks for feedback we will fix it 😄